pkg/ipmi/ipmi.go

```diff
@@ -140,8 +137,8 @@ func (e *Event) marshall() ([]byte, error) {
 		copy(data[3:16], buf.Bytes()[16:29])
 	}
 
-	// OEM non-timestamped
-	if buf.Bytes()[2] >= 0xE0 && buf.Bytes()[2] <= 0xFF {
```

Comment:
I wonder if that kind of test comes from C assumptions about promotion to int (the usual problem with C). Interesting. it's a trap we all fall into from time to time ...

Reply:
I was puzzled as well.